starter/atm.html

```diff
@@ -11,18 +11,18 @@
       <div><img src="ga.png" /></div>
       <div class="title">Bank of GA</div>
     </div>
-    
+
     <div class="account">
       <h2>Checking</h2>
-      <div id= "checkingBalance" class="balance">$0</div>
+      <div id="check" class="balance">$<span id= "checkingBalance">0</span></div>
```

Comment:
Smart move. Only feedback is to get rid of spaces in your attributes:

`id="checkingBalance"` vs `id= "checkingBalance"`